nixos/modules/services/web-apps/healthchecks.nix

```diff
@@ -109,9 +116,11 @@ in
           have support for a `_FILE` variant, run:
           - `nix-instantiate --eval --expr '(import <nixpkgs> {}).healthchecks.secrets'`
           - or `nix eval 'nixpkgs#healthchecks.secrets'` if the flake support has been enabled.
+
+        If the same variable is set in both `settings` and `settingsFile` the value from `settingsFile` has priority.
       '';
       type = types.submodule (settings: {
-        freeformType = types.attrsOf types.str;
+        freeformType = types.attrsOf (types.nullOr types.str);
```

Comment:
This should also already allow {} which is a bit better than null.

Reply:
This is a change that I proposed in https://github.com/NixOS/nixpkgs/pull/314173#discussion_r1613159749
see the comment before it for reason why

Reply:
Removed the change as suggested.
The script issue from [#314173 (comment)](https://github.com/NixOS/nixpkgs/pull/314173#discussion_r1613159749) is fixed by line 133 where `settings.SECRET_KEY_FILE` can still be set as null.